content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -10,27 +10,43 @@ A `rippled` server running in validator mode does everything a stock server does
 
 What makes a validator _different_ is that it also issues validation messages, which are sets of candidate transactions for evaluation by the XRP Ledger network during the [consensus process](consensus-principles-and-rules.html#how-consensus-works).
 
-It's important to understand that these validation messages are used only by `rippled` servers that explicitly trust your validator by listing it on their Unique Node Lists (UNLs). Servers that do not include your validator in their UNLs ignore your validator's messages in the consensus process. A validator that is not included in any UNL is called an _untrusted_ validator.
+It's important to understand that these validation messages are used only by `rippled` servers that explicitly trust your validator by listing it on their Unique Node Lists (UNLs). Once your validator has been added to a UNL, it is a _trusted_ validator that participates in the consensus process. Servers that do not include your validator in their UNLs ignore your validator's messages in the consensus process.
```

Comment:
> It's important to understand that these validation messages are used only by `rippled` servers that explicitly trust your validator by listing it on their Unique Node Lists (UNLs). Once your validator has been added to a UNL, it is a _trusted_ validator that participates in the consensus process.

This is a very tricky part of consensus that's frequently misunderstood. I'd suggest rephrasing it a little to try and head off the misunderstanding that others trusting you changes your server's behavior. 

Also, the previous wording conveniently set up a duality between trusted and untrusted validators (before introducing those terms) in the first couple sentences, which has been made a little harder to catch in this version because of the way it's split up.

Here's a suggested rephrasing:

> It's important to understand that merely sending validation messages does not automatically give you a say in the consensus process. Other servers ignore your validation messages unless your server is on their Unique Node List (UNL). If your server is included in a UNL, it is a _trusted_ validator and its proposals are considered as part of consensus by the servers that trust it.

There's probably room for more improvement there.